broker/src/twilio/twilio_pbx.erl

```diff
@@ -68,7 +68,8 @@ sound_path_for(Name, ?PBX(_)) ->
 sound_quality(?PBX(_)) ->
   "44100".
 
-dial(_Channel, Number, CallerId, ?PBX(Pid)) ->
+dial(_Channel, Number, CallerId, _LocalFilename, _AsteriskFilename, ?PBX(Pid)) ->
+  % FIXME: implement session recording
```

Comment:
Should we log using `lager:warning` at least here? Or is there a way to store in the activity logs that the user can see?

Reply:
I was planning to implement it soon enough that no-one would notice it.

I'm not sure whether understanding how to log this would imply more or less effort than just implementing the feature in Twilio.